src/languages/php/tuning.md

```diff
@@ -0,0 +1,110 @@
+# Performance tuning PHP
+
+Once your application is up and running it still needs to be kept fast.  Platform.sh offers a wide degree of flexibility in how PHP behaves, but that does mean you may need to take a few steps to ensure your site is running optimally.
+
+The following recommendations are guidelines only.  They're also listed in approximately the order we recommend investigating them, although your mileage may vary.
+
+## Upgrade to PHP 7.1+
+
+There is very little purpose to trying to optimize a PHP application on PHP 5.  PHP 7 is generally twice as fast and uses half as much memory as PHP 5, making it unquestionably the first step to take when trying to make a PHP-based site run faster.
+
+To change your PHP version, simply change the `type` key in your `.platform.app.yaml` to the desired PHP version.  As always, test it on a branch first before merging to `master`.
+
+## Ensure that the router cache is properly configured
+
+Although not PHP-specific, a common source of performance issues is a misconfigured cache.  The most common issue is not whitelisting session cookies, which results in a site with any cookies at all, including from analytics tools, never being cached.  See the [router cache](/configuration/routes/cache.md) documentation, and the [cookie entry](/configuration/routes/cache.md#cookies) specifically.
```

Comment:
Maybe mention the cache settings of the `.platform.app.yaml` for static assets too?

Reply:
Added.